frontend/src/lib/keybind.ts

```diff
@@ -0,0 +1,246 @@
+/**
+ * Enum to represent the key strings provided by {@link KeyboardEvent}
+ */
+export enum Keys {
+  J = 'j',
+  D = 'd',
+  K = 'k',
+  H = 'h',
+  V = 'v',
+  T = 't',
+  M = 'm',
+  L = 'l',
+  O = 'o',
+  S = 's',
+  SINGLE_QUOTE = "'",
+  FORWARD_SLASH = '/',
+  CTRL = 'Control',
+  ENTER = 'Enter',
+  ARROW_UP = 'ArrowUp',
+  ARROW_DOWN = 'ArrowDown',
+}
```

Comment:
I think it's a bit more readable to have this as a type instead of an enum

```ts
export const Keys = [
  'j',
  'd',
  'k',
  'h',
  'v',
  't',
  'm',
  'l',
  'o',
  's',
  "'",
  '/',
  'Control',
  'Enter',
  'ArrowUp',
  'ArrowDown',
];
export type Key = (typeof Keys)[number];
```